src/main/utils/jsonl.ts

```
@@ -244,7 +284,10 @@ export function calculateMetrics(messages: ParsedMessage[]): SessionMetrics {
     }
   }
 
-  for (const msg of messages) {
+  // Calculate cost per-message, then sum (tiered pricing applies per-API-call, not to aggregated totals)
+  let costUsd = 0;
```

ESLint

[error] 'costUsd' is never reassigned. Use 'const' instead.

(prefer-const)